platform/darwin/src/MGLMapSnapshotter.mm

```diff
@@ -62,14 +63,16 @@ - (instancetype)initWithOptions:(MGLMapSnapshotOptions*)options;
         _loading = false;
         
         mbgl::DefaultFileSource *mbglFileSource = [MGLOfflineStorage sharedOfflineStorage].mbglFileSource;
-        mbglThreadPool = mbgl::sharedThreadPool();
+        _mbglThreadPool = mbgl::sharedThreadPool();
         
         std::string styleURL = std::string([options.styleURL.absoluteString UTF8String]);
         
         // Size; taking into account the minimum texture size for OpenGL ES
+        // For non retina screens the ratio is 1:1 MGLSnapshotterMinimumPixelSize
+        
         mbgl::Size size = {
-            static_cast<uint32_t>(MAX(options.size.width, 64)),
-            static_cast<uint32_t>(MAX(options.size.height, 64))
+            static_cast<uint32_t>(MAX(options.size.width, MGLSnapshotterMinimumPixelSize/options.scale)),
```

Comment:
If the developer sets a scale other than the device’s native scale, does the effective minimum pixel size change?